tests/cypress/integration/reportRepair/communal.spec.js

```diff
@@ -60,7 +60,7 @@ describe('communal', () => {
           makeSelectionAndClickButton('No');
         });
         it('should redirect them to postcode then address page respectively',  () => {
-          cy.url().should('include', '/report-repair/postcode');
+          cy.url().should('include', '/report-repair/tenant-or-leaseholder');
```

Comment:
Are we leaving just tests for the full version or should there be an MVP test file to test that it goes to the postcode page?

Reply:
Extracted to `communalMvp.spec.js`